layouts/partials/head.html

```diff
@@ -48,23 +48,5 @@
 
   <link rel="shortcut icon" href="{{ .Site.BaseURL }}favicon.ico" />
   <meta name="theme-color" content="#e3ddeef2" />
-  <link rel="manifest" href="manifest.json" />
-  <script src="{{ .Site.BaseURL }}sw.js"></script>
-  <script>
-    if ("serviceWorker" in navigator) {
-      window.addEventListener("load", function () {
-        navigator.serviceWorker.register("/sw.js").then(
```

Comment:
Do we also want to remove `static/sw.js` from the repo until we consciously re-add this functionality?